### DISCUSSION TITLE
Bundle size keeps growing

### DISCUSSION THREAD

**Discussion by b3nk3** [algolia/instantsearch#6269] [Q&A]
Hey there,

We've noticed a massive jump in bundle size since the first release of version 7 to the latest, but especially going from v7.8 to v7.9.

Any idea why is this?

**Comment by Haroenv** [comment]
The main recent bundle size increase is because of the inclusion of Recommend. We're planning on a major version in the future that can remove a lot of deprecated features, but it takes some time to plan to do that correctly. If you're using bundlesize to measure as well, that's not the most representative measure, as you usually don't use all the widgets, so should have less impact of new features.
